static/docs/tutorial/define-ml-pipeline.md

```diff
@@ -11,9 +11,9 @@ files and download a 40MB data archive into this directory.
 ### Expand to learn how to download on Windows
 
 Windows doesn't include the `wget` utility by default, so you'll need to use the
-browser to download `data.xml`. Save it into the `data` subdirectory.
+browser to download `data.xml`. Save it into the `data/` subdirectory.
 (Right-click [this link](https://data.dvc.org/tutorial/ver/data.zip) and click
-`Save link as`(Chrome) or `Save object as`(Firefox)).
+`Save Link As...`(Chrome).
```

Comment:
why removing the Firefox version? :)

Reply:
I couldn't find the exact text that the Firefox context menu has corresponding to "Save Link As...". I was going to install it to check and then I realized FF has fallen to under 10% global browser usage. Google Chrome is totally dominating the market! So I decided it's not really worth pointing out FF (over other small options at least). https://www.w3schools.com/browsers/default.asp

But lmk if you prefer it back and I'll find out the exact current text for this function.

Reply:
👍